common/changes/office-ui-fabric-react/remove-input-bottom-margin_2017-05-31-16-01.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Inputs: removed bottom margin causing issues when putting more than one per line",
+      "type": "minor"
```

Comment:
Yeah...I can put I in todays' newsletter to watch out for it. But then again, React is on 15, or 16, or is it 200 by now?